### DISCUSSION TITLE
The Office of Readings

### DISCUSSION THREAD

**Discussion by MrKlopp** [DivinumOfficium/divinum-officium#4229] [Q&A] [answered]
Is there any chance you could integrate the post Vatican Office of Readings as an addition or otherwise as a comparison to the traditional Matins please?. I would consider that the Office of Readings the only worthy aspect of the reform and would regard this as separate to the traditional hours which secure the obligations  of the Psalter - unlike that of the modern Liturgy of the Hours.
I believe you integrated the Novus Ordo Mass as a comparable, which I found useful. I would be grateful if you could include the Office too.

**Comment by APMarcello3** [comment, marked as answer]
As the stated aim of this Project encompasses solely the liturgical texts in usage prior to the reforms of 1969, providing the Office of Readings would be outside of the scope of Divinum Officium. Other websites/apps such as Universalis and iBreviary are available and include these texts.
